### DISCUSSION TITLE
File Uploading Error

### DISCUSSION THREAD

**Discussion by ArthurPatriot** [livewire/livewire#3190] [7. Bugs (archived)]
### Discussed in https://github.com/livewire/livewire/discussions/3172

<div type='discussions-op-text'>

<sup>Originally posted by **ArthurPatriot** June 22, 2021</sup>
```
ErrorException
is_file(): open_basedir restriction in effect. File(/tmp/phpjyAc5g) is not within the allowed path(s): (/home/test/web/***/public_html:/home/test/tmp)
```

Changing files temp dir in livewire config not help...</div>

**Comment by jeremiatarigan** [comment]
Did you already solve this issue ?
